File: bindings/cashu-sdk-ffi/bindings-python/examples/wallet.py
from cashu_sdk import Wallet, Client, Amount

client = Client("https://mutinynet-cashu.thesimplekid.space")

mint_keys = client.get_keys()

wallet = Wallet(client, mint_keys)

mint_request = wallet.request_mint(Amount.from_sat(10))

print(mint_request.invoice())

print(mint_request.hash())

File: bindings/cashu-sdk-js/examples/wallet.js
const {Amount, loadWasmAsync, Wallet, Client } = require("..");


async  function main() {
  await loadWasmAsync();

  let client = new Client("https://mutinynet-cashu.thesimpekid.space");

  let keys = await client.getKeys();

  let wallet = new Wallet(client, keys);

  let amount = Amount.fromSat(BigInt(10));
  let pr = await wallet.requestMint(amount);

  console.log(pr);
}

main();
